### PR TITLE
Wallarm Rules via Wallarm Provider

### PR DESCRIPTION
**Pull request by 416e64726579**
I've included the configuration of the rules in Wallarm Cloud via Terraform Wallarm Provider.

Please, have a glance and check out on your test account.

### CODE REVIEW

File: terraform/wallarm-rules/main.tf [hunk 1]
```diff
@@ -0,0 +1,252 @@
+# 
+# Define provider parameters
+# 
+provider "wallarm" {
```
Comment (vgartvich-wallarm) [inline terraform/wallarm-rules/main.tf:4]: How to define Wallarm customer account ID?

File: terraform/wallarm-rules/main.tf [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+# 
+# Virtual Patch section
+# 
+resource "wallarm_rule_vpatch" "splunk" {
+  attack_type =  ["sqli", "nosqli"]
+
+  action {
+    type = "iequal"
+    value = "splunk.wallarm-demo.com:88"
```
Comment (vgartvich-wallarm) [inline terraform/wallarm-rules/main.tf]: Please take the "wallarm-demo.com" domain name from TF variable in this and several other places in the PR.

File: terraform/wallarm-rules/main.tf (hunk 1 above)
Comment (416e64726579) [reply terraform/wallarm-rules/main.tf:4]: client_id = var.wallarm_client_id

Comment (416e64726579) [reply terraform/wallarm-rules/main.tf:4]: `client_id = var.wallarm_client_id`
It is possible to define this parameter per resource.

**Pull request merged**